src/overlays/actors/ovl_En_Po_Fusen/z_en_po_fusen.c

```diff
@@ -57,44 +56,49 @@ static ColliderSphereInit sSphereInit = {
     { 0, { { 0, 0, 0 }, 200 }, 100 },
 };
 
+typedef enum {
+    /* 0x0 */ POFUSEN_DMGEFF_NONE,
+    /* 0xF */ POFUSEN_DMGEFF_POP = 0xF,
+} PoFusenDamageEffect;
```

Comment:
```suggestion
typedef enum {
    /* 0x0 */ POE_BALLOON_DMGEFF_NONE,
    /* 0xF */ POE_BALLOON_DMGEFF_POP = 0xF,
} PoeBalloonDamageEffect;
```
Since you use `POE_BALLOON_LIMB_XXX` in the object file, might be nice to make it consistent. Same would apply to the params